### DISCUSSION TITLE
Is it possible to open a directory just for the purpose of viewing it's contents?

### DISCUSSION THREAD

**Discussion by strangeways-dev** [yasirkula/UnityNativeFilePicker#34] [Q&A] [answered]
Is it possible to open a directory just for the purpose of viewing it's contents?
My app allows users to generate NFT's and I want them to be able to click a button to open the directory to the generated NFT collection so they can view what they've generated. Obviously I could just use the file picker and ignore the returned paths but it won't make for a good user experience.
Thanks in advance!

**Comment by yasirkula** [comment, marked as answer]
I see. This is unfortunately not possible with NativeFilePicker. On Android, this might prove useful: https://stackoverflow.com/questions/17165972/android-how-to-open-a-specific-folder-via-intent-and-show-its-content-in-a-file. On iOS, I unfortunately have no idea 🙃

**Reply by strangeways-dev** [reply]
Perfect! Thank you! Fortunately that won't be a problem because I don't intend to use IOS, android is more than enough pain for me to deal with atm lol. I really hope my app is successful and I will give you a nice donation, your plugins and your advice has helped me out so much!

**Reply by yasirkula** [reply]
Happy to help and thank you for your support :) Wish the best with your app!
